### ISSUE TITLE
Can't access my Pixel through USB

### ISSUE THREAD

**Issue by DUOLabs333** [utmapp/UTM#5451]
**Describe the issue**  
I tried to connect my phone to my Macbook using a USB-C cable for ADB. However, when I did, UTM just prompted over and over again to connect. If I click "Don't show this again", nothing shows up.

**Configuration**  
* UTM Version: 4.2.5
* macOS Version: 11
* Mac Chip (Intel, M1, ...): M1

**Crash log**  
If the app crashed, you need a crash log. To get your crash log, open Console.app, go to `Crash Reports`, and find the latest entry for either UTM, QEMU, QEMUHelper, or qemu-\*. Right click and choose `Reveal in Finder`. Attach the report here.

**Debug log**  

**Upload VM**

**Comment by DUOLabs333** [comment]
Ok, after some testing, it seems that this does work, but only after pressing "Confirm" a lot. This feels similar to #5429. I'm not sure why it only happens to Android phones though (this does not happen to regular USB devices).

**Comment by DUOLabs333** [comment]
Actually, maybe not. #5249 is when the action occurs, but the dialog still pops up several times. However, in this case, in the guest's `dmesg`, I can see it being repeatedly disconnected (in fact, 8). However, I must have pressed "Confirm" at least 20 times, so it's probably a mixture of issues.

**Comment by herve-ch** [comment]
i had the same problem, in my case it was the usb debug mode that was activated in my phone. disabling it makes the usb passthrough working nicely again!

**Comment by herve-ch** [comment]
@DUOLabs333 why use UTM for adb? it works on native macOS, you can install adb and fastboot with homebrew. Or maybe you develop some android app?

**Comment by DUOLabs333** [comment]
I wrote my own replacement for SPICE's usb-redir, and no longer have this issue (although, for some reason, my Pixel shows up multiple times under `lsusb` if PTP/MTP is enabled on the phone).

**Comment by osy** [comment]
Do you think we should integrate your replacement with UTM? If so feel free to open a new issue or pull request. Thanks.

**Comment by DUOLabs333** [comment]
Eh, not really --- while it works well for me, it probably won't work well with the rest of UTM.
